modules/ROOT/pages/associate-environments.adoc

```diff
@@ -5,15 +5,60 @@ endif::[]
 
 Before deploying applications or API gateways to Anypoint Runtime Fabric, you must associate a Runtime Fabric with one or more Anypoint Platform environments. When deploying a Mule application, you cannot select a Runtime Fabric as a deployment target until you have performed this step in the environment where you want to deploy.
 
+== Associate Business Groups
+
+You can share Runtime Fabric with an organization's business groups. This allows other business groups to deploy applications to a single Runtime Fabric. To use this feature, the business groups must fall under the scope of the organization the Runtime Fabric is registered to.
+
+To associate a Runtime Fabric across business groups:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select the *Runtime Fabric* tab on the left sidebar.
+. Select the Runtime Fabric to manage.
+. On the *Environments* tab, click *Add business group*.
+. Select the business groups from the drop-down list:
+
+** *This business group and its children*
++ 
+This option allows you to select either all environments, all sandbox environment or all production environments for the selected business group and all its children.
+
+** *Selected business groups* 
++
+Enter the business group name in the *Add business group* field and select it from the list. This option allows you to select specific environments for specific child business groups.
```

Comment:
`select specific environments for specific child business groups` -> You can select specific environments for root org as well as it's children. So, can we change this to `specific business groups`

Reply:
Changed to:

_This option allows you to select specific environments for root organizations as well as it's children._

Let me know if this is clear. Also, please indicate which other sentences need to be updated with this clarification.